src/scene_title.cpp

```diff
@@ -29,15 +30,17 @@
 #include "transition.h"
 #include "input.h"
 #include "main_data.h"
+#include "meta.h"
 #include "output.h"
 #include "player.h"
 #include "scene_battle.h"
+#include "scene_import.h"
 #include "scene_load.h"
 #include "scene_map.h"
 #include "window_command.h"
 #include "baseui.h"
 
-Scene_Title::Scene_Title() {
+Scene_Title::Scene_Title() : new_game_index(0), continue_index(1), exit_index(2), import_index(-1) {
```

Comment:
because we use C++11 you can assign them directly in the Header instaed of using this initializer